### PR TITLE
Remove caching around SessionService calls.

### PR DESCRIPTION
**Pull request by wtritch**
I've removed all caching around calls to the SessionService.  I've left in caching around the AssessmentService, StudentService, and ConfigService.

### CODE REVIEW

File: service/src/main/java/tds/exam/services/impl/SessionServiceImpl.java [hunk 1]
```diff
@@ -61,7 +57,6 @@ public SessionServiceImpl(final RestTemplate restTemplate, final ExamServiceProp
     }
 
     @Override
-    @Cacheable(CacheType.LONG_TERM)
```
Comment (millerts) [inline service/src/main/java/tds/exam/services/impl/SessionServiceImpl.java]: This can stay long term.  I believe this should probably be in ConfigsService long term, but for now it resides in Session service.  The configuration is static config data that doesn't change once configured.  However, that lives in session schema :/ which is why it is in the session service.

TLDR - Leave it long term for now

Comment (wtritch) [inline service/src/main/java/tds/exam/services/impl/SessionServiceImpl.java]: done

File: scoring/pom.xml [hunk 2]
```diff
@@ -20,14 +20,13 @@
...
         <item-renderer.version>3.0.9</item-renderer.version>
-        <tds-student-common.version>3.0.4</tds-student-common.version>
     </properties>
 
     <dependencies>
         <dependency>
             <groupId>org.opentestsystem.delivery</groupId>
             <artifactId>student-library</artifactId>
-            <version>${tds-student-common.version}</version>
+            <version>${tds-student-library.version}</version>
```
Comment (wtritch) [inline scoring/pom.xml]: Fixed a place where we were double-declaring the student-common version in ExamService.
